README.md

```diff
@@ -91,19 +103,74 @@ See [example docker-compose.yml](docker-compose.yml).
 
 ## Run RCON commands
 
-Open a shell into your container via `docker exec -ti palworld-dedicated-server bash`, then you can run commands against the gameserver via the command `rcon` or `rconcli`
+Open a shell into your container via `docker exec -ti palworld-dedicated-server bash`, then you can run commands against the gameserver via the command `rconcli`
 
 ```shell
-$:~/steamcmd$ rcon showplayers
+$:~/steamcmd$ rconcli showplayers
 name,playeruid,steamid
-$:~/steamcmd$ rcon info
+$:~/steamcmd$ rconcli info
 Welcome to Pal Server[v0.1.3.0] jammsen-docker-generated-20384
-$:~/steamcmd$ rcon save
+$:~/steamcmd$ rconcli save
+Complete Save
+```
+
+You can also use `docker exec palworld-dedicated-server rconcli <command>` right on your terminal/shell.
+
+```shell
+$ docker exec palworld-dedicated-server rconcli showplayers
+name,playeruid,steamid
+
+$ docker exec palworld-dedicated-server rconcli info
+Welcome to Pal Server[v0.1.3.0] jammsen-docker-generated-20384
+
+$ docker exec palworld-dedicated-server rconcli save
 Complete Save
 ```
 
 > **Important:** Please research the RCON-Commands on the official source: https://tech.palworldgame.com/server-commands
 
+## Backup Manager
+
+Usage: `docker exec palworld-dedicated-server backup [command] [arguments]`
+
+| Command | Argument           | Required/Optional | Default Value                     | Values           | Description                                                                                                                                                                          |
+| ------- | ------------------ | ----------------- | --------------------------------- | ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| create  | N/A                | N/A               | N/A                               | N/A              | Creates a backup.                                                                                                                                                                    |
+| list    | `<number_to_list>` | Optional          | N/A                               | Positive numbers | Lists all backups.<br>If `<number_to_list>` is specified, only the most<br>recent `<number_to_list>` backups are listed.                                                             |
+| clean   | `<number_to_keep>` | Optional          | `BACKUP_RETENTION_AMOUNT_TO_KEEP` | Positive numbers | Cleans up backups.<br>If `<number_to_list>` is specified, cleans and keeps<br>the most recent`<number_to_keep>` backups.<br>If not, default to `BACKUP_RETENTION_AMOUNT_TO_KEEP` var |
+
+Examples:
+
+```shell
+$ docker exec palworld-dedicated-server backup
+>> Backup 'saved-20240203_032855.tar.gz' created successfully.
```

Comment:
I do `>>> abc` for a new category and `> abc` for small info feedback inside that category, never did `>>` though